extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -33,7 +35,13 @@ var WikiaSearch = {
 			}
 		});
 
+		$('.result-link').on('click', function(event) {
+			console.log(event.target);
```

Comment:
So I see we're doing a console.log here, but not in other places where we're calling into the search tracking module. Eg, there's no console.log for `trackSearchResultsImpression`. Is there a reason to log this and not others? Or a reason to log this at all?

Reply:
this is just leftover I forgot to remove :)